README.md

```diff
@@ -23,81 +24,97 @@ This module adds a provider to manage IIS sites and application pools.
 
 ### Beginning with puppetlabs-iis
 
-This module can both manage and install IIS on your server. For example, a minimal IIS install can be accomplished by ensuring the `Web-WebServer` & `Web-Scripting-Tools` Windows Features are present.
+This module can both manage and install IIS on your server. For example, a minimal IIS install can be accomplished by ensuring the `Web-WebServer` and `Web-Scripting-Tools` Windows Features are present.
 
 Here is an example that installs IIS and creates a web site using the default application pool.
 
 ```puppet
 $iis_features = ['Web-WebServer','Web-Scripting-Tools']
 
 iis_feature { $iis_features:
-  ensure => present,
+  ensure => 'present',
 } ->
 
 iis_site { 'minimal':
   ensure          => 'started',
   physicalpath    => 'c:\\inetpub\\minimal',
-  applicationpool => 'DefaultApplicationPool',
+  applicationpool => 'DefaultAppPool',
+  require         => File['minimal'],
+}
+
+file { 'minimal':
+  ensure => 'directory',
+  path   => 'c:\\inetpub\\minimal',
 }
 ```
 
 ## Usage
 
-This example will create and configure an IIS `web site` in the `Started` state with the physical path set, which makes use of an IIS `application pool` named `minimal_site_app_pool` in the `Started` state.
+This minimal example will create a web site named 'complete' using an application pool named 'minimal_site_app_pool'.
 
 ```puppet
 iis_application_pool { 'minimal_site_app_pool':
   ensure                  => 'present',
+  state                   => 'started'
   managed_pipeline_mode   => 'Integrated',
   managed_runtime_version => 'v4.0',
-  state                   => 'Started'
 } ->
 
 iis_site { 'minimal':
   ensure          => 'started',
   physicalpath    => 'c:\\inetpub\\minimal',
   applicationpool => 'minimal_site_app_pool',
+  require         => File['minimal'],
+}
+
+file { 'minimal':
+  ensure => 'directory',
+  path   => 'c:\\inetpub\\minimal',
 }
 ```
 
-This complete example will create and configure an IIS Site called 'complete' in the `Started` state with the physical path set, which makes use of an IIS Application Pool named 'complete_site_app_pool' in the `Started` state. This example also uses HTTPS, sets the certificate information and creates a virtual directory called 'vdir' in the site.
+This complete example will create a web site named 'complete' using an application pool named 'complete_site_app_pool', with a virtual directory named 'vdir'. This example uses the `puppetlabs-acl module` to set permissions on directories.
 
 ```puppet
-# Create Directory Structure
+# Create Directories
+
 file { 'c:\\inetpub\\complete':
   ensure => 'directory'
 }
+
 file { 'c:\\inetpub\\complete_vdir':
   ensure => 'directory'
 }
 
-# This example makes use of the puppetlabs-acl module to set the permissions on the
-# the web site directories
```

Comment:
I didn't delete that comment, I moved it out of the code and into the "This complete example ..." paragraph above @ Line 76. Is that okay, or should I move it back?